langs/tree-sitter-langs-build.el

```diff
@@ -85,6 +85,7 @@ If VERSION and OS are not specified, use the defaults of
     (c-sharp    "075a1b2")
     (cpp        "5e7476b")
     (css        "23f2cb9")
+    (elm        nil nil "https://github.com/razzeee/tree-sitter-elm")
```

Comment:
We should specify a git revision here.

Reply:
Ok - I just chose the current master. I though tleaving it out would go for `origin/master` by default.

Reply:
Yes, it would take `origin/master` by the default, so we should specify a commit hash, to make it reproducible.

Reply:
Makes sense